tail.go

```diff
@@ -265,35 +270,29 @@ func (tail *Tail) tailFileSync() {
 			}
 		}
 
-		line, err := tail.readLine()
+		line, llen, err := tail.readLine()
+		noffset := uint64(offset) + llen
 
 		// Process `line` even if err is EOF.
 		if err == nil {
-			cooloff := !tail.sendLine(line)
-			if cooloff {
-				// Wait a second before seeking till the end of
-				// file when rate limit is reached.
-				msg := ("Too much log activity; waiting a second " +
-					"before resuming tailing")
-				tail.Lines <- &Line{msg, time.Now(), errors.New(msg)}
-				select {
-				case <-time.After(time.Second):
-				case <-tail.Dying():
-					return
-				}
-				if err := tail.seekEnd(); err != nil {
-					tail.Kill(err)
-					return
-				}
-			}
+			_ = !tail.sendLine(line, noffset)
+			// if cooloff {
+			// 	// Wait a second before seeking till the end of
+			// 	// file when rate limit is reached.
+			// 	msg := ("Too much log activity; waiting a second " +
+			// 		"before resuming tailing")
+			// 	tail.Lines <- &Line{msg, time.Now(), errors.New(msg), 0}
+			// 	select {
+			// 	case <-time.After(time.Second):
+			// 	case <-tail.Dying():
+			// 		return
+			// 	}
+			// 	if err := tail.seekEnd(); err != nil {
+			// 		tail.Kill(err)
+			// 		return
+			// 	}
+			// }
 		} else if err == io.EOF {
-			if !tail.Follow {
-				if line != "" {
-					tail.sendLine(line)
-				}
-				return
-			}
-
 			if tail.Follow && line != "" {
```

Comment:
why is this removed?

Reply:
nothing specific. we never hit this condition.